src/topp/QualityControl.cpp

```diff
@@ -420,7 +430,47 @@ class TOPPQualityControl : public TOPPBase
 
       if (export_evidence.isValid())
       {
-        export_evidence.exportFeatureMap(*fmap,cmap);
+        auto& cmapProtIds = cmap.getProteinIdentifications();
+        map<String,String> fastaMap {};
+        //map the identifier to the description so that we can access the description via the cmap-identifier
+        for(const auto& entry : protDescription)
+        {
+          fastaMap.emplace(entry.identifier, entry.description);
+        }
+
+        if(cmapProtIds.empty())
+        {
+          OPENMS_LOG_ERROR << "No ProteinIdentifications in consensusmap... aboarting" << std::endl;
```

Comment:
you are not really aborting here...

Just say that protein names and gene names will not be available in the output (or better yet, do that when you actually write the columns, so the information is closer to the place where the output actually happens).